src/network/gamehost.cc

```diff
@@ -1276,10 +1276,7 @@ void GameHost::set_player_ai(uint8_t number, const std::string& name, bool const
 
 	// Broadcast changes
 	SendPacket packet;
```

Comment:
Delete this variable. Same for all instances where you call broadcast_setting_player.